client/hosting/performance/site-performance.tsx

```diff
@@ -300,7 +302,9 @@ export const SitePerformance = () => {
 			disabled={ disableControls }
 			onChange={ ( page_id ) => {
 				const url = new URL( window.location.href );
-
+				recordTracksEvent( 'calypso_performance_profiler_page_selector_change', {
+					page: pages.find( ( page ) => page.value === page_id )?.label ?? '/',
```

Comment:
How useful is it to record the page title? It's not really aggregatable data.
I feel like a `is_home` event prop would make more sense than the raw page title which we can't really do anything with in the tracks tool.

Reply:
And I think we should add the `version` prop here too.

Reply:
I was recording the `page_id` at first but that didn't seem to be useful either.